es_ES/flutter-for-ios.md

```diff
@@ -0,0 +1,1937 @@
+los `Text` widget:
+
+<!-- skip -->
+{% prettify dart %}
+body: Center(
+  child: TextField(
+    decoration: InputDecoration(hintText: "Esto es una pista"),
+  ),
+)
+{% endprettify %}
+
+## ¿Cómo puedo mostrar los errores de validación?
+
+Al igual que lo harías con una "pista", pasa un objeto `InputDecoration`
+al constructor de decoración para el `Text` widget.
+
+Sin embargo, no desea comenzar mostrando un error.
+En su lugar, cuando el usuario haya introducido datos no válidos, actualice el estado y pase un nuevo objeto `InputDecoration`.
+
+<!-- skip -->
+{% prettify dart %}
+class SampleApp extends StatelessWidget {
+  // Este widget es la raíz de su aplicación.
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      title: 'Sample App',
+      theme: ThemeData(
+        primarySwatch: Colors.blue,
+      ),
+      home: SampleAppPage(),
+    );
+  }
+}
+
+class SampleAppPage extends StatefulWidget {
+  SampleAppPage({Key key}) : super(key: key);
+
+  @override
+  _SampleAppPageState createState() => _SampleAppPageState();
+}
+
+class _SampleAppPageState extends State<SampleAppPage> {
+  String _errorText;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("Sample App"),
+      ),
+      body: Center(
+        child: TextField(
+          onSubmitted: (String text) {
+            setState(() {
+              if (!isEmail(text)) {
+                _errorText = 'Error: Esto no es un correo electrónico';
+              } else {
+                _errorText = null;
+              }
+            });
+          },
+          decoration: InputDecoration(hintText: "Esto es una pista", errorText: _getErrorText()),
+        ),
+      ),
+    );
+  }
+
+  _getErrorText() {
+    return _errorText;
+  }
+
+  bool isEmail(String em) {
+    String emailRegexp =
+        r'^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$';
+
+    RegExp regExp = RegExp(p);
+
+    return regExp.hasMatch(em);
+  }
+}
+{% endprettify %}
+
+# Interactuar con el hardware, servicios de terceros y con la plataforma
+
+## ¿Cómo interactúo con la plataforma y con el código nativo de la plataforma?
+
+Flutter no ejecuta código directamente en la plataforma subyacente; más bien, el código Dart que constituye una aplicación Flutter se ejecuta de forma nativa en el dispositivo, " evitando " el SDK proporcionado por la plataforma. Esto significa, por ejemplo, que cuando realizas una solicitud de red en Dart, se ejecuta directamente en el contexto de Dart. No se utilizan las API de Android o iOS que normalmente se aprovechan al escribir aplicaciones nativas. Tu aplicación Flutter sigue alojada en una aplicación nativa
```

Comment:
Your Flutter app is still hosted in a native app’s ViewController as a view, but you don’t have direct access to the ViewController itself, or the native framework

Tu app Flutter sigue alojada en el ViewController de una app nativa, como un vista, pero no tienes acceso directo al `ViewController`en si mismo, ni al framework nativo.